ivadomed/scripts/curate_microscopy_data_demo.py

```diff
@@ -0,0 +1,215 @@
+"""
+The data are structure in a folder such as:
+demo_microscopy_dataset/
+├── 20220309_demo_mouse_07_0001
+│         ├── image.png
+│         ├── image_seg-axon.png
+│         ├── image_seg-axonmyelin.png
+│         └── image_seg-myelin.png
+├── 20220309_demo_mouse_07_0002
+│         ├── image.png
+│         ├── image_seg-axon.png
+│         ├── image_seg-axonmyelin.png
+│         └── image_seg-myelin.png
+└── 20220309_demo_mouse_07_0003
+          ├── image.png
+          ├── image_seg-axon.png
+          ├── image_seg-axonmyelin.png
+          └── image_seg-myelin.png
+
+"""
+
+# Define various dependencies
+
+import glob
+import os
+import shutil
+import json
+import argparse
+import subprocess
+import csv
+from textwrap import dedent
+
+## Dictionary for images
+images = {
+    "image.png": "_TEM.png"
+}
+
+## Dictionary for derivatives
+der = {
+    "image_seg-axon.png": "_TEM_seg-axon-manual.png",
+    "image_seg-axonmyelin.png": "_TEM_seg-axonmyelin-manual.png",
+    "image_seg-myelin.png": "_TEM_seg-myelin-manual.png"
+}
+
+# Define function to get the input and output path for data
+def get_parameters():
+    parser = argparse.ArgumentParser(description='This script is curating the demo dataset to BIDS')
+    # Define input path
+    parser.add_argument("-d", "--data",
+                        help="Path to folder containing the dataset to be curated",
+                        required=True)
+    # Define output path
+    parser.add_argument("-o", "--outputdata",
+                        help="Path to output folder",
+                        required=True,
+                        )
+    arguments = parser.parse_args()
+    return arguments
+
+# Define function to create json sidecar
+def create_json_sidecar(output_data, sub_id):
+
+    # Path for each subject  destination bids folder
+    path_folder_sub_id_bids = os.path.join(output_data, sub_id, 'micr')
+
+    # Create filename for json sidecar
+    item_out = sub_id + "_TEM.json"
+
+    # Define json sidecar content
+    data_json = {"PixelSize": [0.00236, 0.00236],
+                 "PixelSizeUnits: "um"
```

Comment:
```suggestion
                 "PixelSizeUnits": "um"
```
Sorry, my mistake, there was a missing quote in my previous suggestion.